### DISCUSSION TITLE
Ignore this, its a demonstration for a YouTube Video #2

### DISCUSSION THREAD

**Discussion by Thinkright20** [community/community#62160] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Please ignore

**Comment by Badges-Tutorial** [comment]
Response #2

**Comment by queenofcorgis** [comment]
@Thinkright20  @Badges-Tutorial  GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
